extensions/amp-list/amp-list.md

```diff
@@ -448,7 +486,11 @@ may make a request to something like `https://foo.com/list.json?0.8390278471201`
 ### `src` (required)
 
 The URL of the remote endpoint that returns the JSON that will be rendered
-within this `<amp-list>`. This must be a CORS HTTP service. The URL's protocol must be HTTPS.
+within this `<amp-list>`. There are three valid protocols for the `src` attribute.
+
+1. **https**: This must refer to a CORS HTTP service. Insecure HTTP is not supported.
+2. **amp-state**: For initializing from `<amp-state>` data. See [Initialization from <amp-state>](#initialization-from-amp-state) for more details.
+3. **amp-script**: For using `<amp-script>` functions as the data source. See [Using <amp-script> as a data source](#using-amp-script-as-a-data-source) for more details.
```

Comment:
Actually, just removing "This is" does make point #2 clearer for me!

Only problem left here is that raw HTML tags in links will actually become real HTML. Just put `<amp-state>` and `<amp-script>` in backticks in the links in #2 and #3, and you'll be all set!

Reply:
> Only problem left here is that raw HTML tags in links will actually become real HTML

Good to know! Fixed